spec/features/state_file/complete_intake_spec.rb

```diff
@@ -612,6 +612,84 @@
 
       expect(page).to have_text I18n.t("state_file.questions.submission_confirmation.edit.title", state_name: "Maryland")
     end
+
+    it "has mfj two income subtractions content", required_schema: "md" do
+      visit "/"
+      click_on "Start Test MD"
+
+      expect(page).to have_text I18n.t("state_file.landing_page.edit.md.title")
+      click_on I18n.t('general.get_started'), id: "firstCta"
+
+      expect(page).to have_text I18n.t("state_file.questions.md_eligibility_filing_status.edit.title", year: MultiTenantService.statefile.current_tax_year)
+      # select optoins that allow us to proceed
+      click_on "Continue"
+
+      # click continue on eligibility & check the messaging is correct here?
+      # select mfj
+      choose I18n.t("general.affirmative")
+      choose I18n.t("general.negative"), id: "state_file_md_eligibility_filing_status_form_eligibility_homebuyer_withdrawal_mfj_no"
+      choose I18n.t("general.negative"), id: "state_file_md_eligibility_filing_status_form_eligibility_home_different_areas_no"
+
+      click_on "Continue"
+
+      click_on "Continue"
+
+      step_through_initial_authentication(contact_preference: :email)
+
+      expect(page).to have_text I18n.t('state_file.questions.terms_and_conditions.edit.title')
+      click_on I18n.t("state_file.questions.terms_and_conditions.edit.accept")
+
+      step_through_df_data_transfer("Transfer Zeus two w2s")
```

Comment:
is it possible to use `Transfer Zeus two w2s` for the fixture in the existing spec?

Reply:
indeed, I've combined both contexts to use `Transfer Zeus two w2s`